fix: use `window.fetch` in `load` functions to allow libraries to patch it (#10009)
sveltejs/kit: .changeset/sharp-birds-invent.md

---
'@sveltejs/kit': patch
---

> fix: Use `window.fetch` instead of unpatched fetch in `load` functions

Reviewer: ```suggestion
fix: use `window.fetch` in `load` functions to allow libraries to patch it
```
